### DISCUSSION TITLE
What do you think of GH Discussions (vs Spectrum?)

### DISCUSSION THREAD

**Discussion by marcdumais-work** [eclipse-theia/theia#7593] [General] [answered]
@svenefftinge requested this new feature be enabled on our repo, and so it was.

I haven't used this yet, so I thought now was a good time :)

**Comment by benoitf** [comment]
let's try :-)

**Comment by akosyakov** [comment, marked as answer]
Let's discuss it on next dev meeting, before deciding what to use and promoting it. Spectrum is going to be deprecated in favour of GitHub Discussions. But GitHub Discussions don't have some features, like direct messages.

**Comment by jankeromnes** [comment]
Also, as mentioned in the dev meeting, I previously made a [script](https://gist.github.com/jankeromnes/dcd9bd8cd1a3df2d32d8541adb7758f2) that can export all threads of a Spectrum community as JSON. This could help ensure that the Theia community's history doesn't get lost one day.

The script can also re-post all the threads to a Discourse instance -- that's what we used to migrate all history from Gitpod's Spectrum over to https://community.gitpod.io (a self-hosted Discourse instance that we're quite happy with).

If you're curious to try a Discourse instance for Theia, I'm happy to set one up, just as an experiment to compare GitHub Discussions and Discourse. But if Discussions becomes the new official community "place" for Theia, I think my script can easily be rewritten to upload all old Spectrum threads to here as well (provided there is an API for that).

**Reply by akosyakov** [reply]
@jankeromnes I don't think Discussions will replace the community. Setting up Discourse sounds great especially if we can reuse your migration scripts.
